build-logic/api-extractor/src/main/java/org/gradle/internal/tools/api/impl/JavaApiMemberWriter.java

```diff
@@ -69,15 +79,17 @@ protected void writeMethod(MethodMember method) {
         mv.visitEnd();
     }
 
-    protected void writeClassAnnotations(Set<AnnotationMember> annotationMembers) {
+    @Override
+    public void writeClassAnnotations(Set<AnnotationMember> annotationMembers) {
         for (AnnotationMember annotation : annotationMembers) {
             AnnotationVisitor annotationVisitor =
                 apiMemberAdapter.visitAnnotation(annotation.getName(), annotation.isVisible());
             writeAnnotationValues(annotation, annotationVisitor);
         }
     }
 
-    private void writeMethodAnnotations(MethodVisitor mv, Set<AnnotationMember> annotationMembers) {
+    @Override
+    public void writeMethodAnnotations(MethodVisitor mv, Set<AnnotationMember> annotationMembers) {
```

Comment:
Why make these public? It doesn't look like any of the code in this PR calls it. Do we foresee needing to call it in the future?

Reply:
These are `ApiMemberWriter`'s methods being implemented.